$OpenBSD: patch-data_ui_epiphany-ui_xml,v 1.1 2011/10/17 16:56:28 jasper Exp $

Remove non-functioning and confusing button untill the underlying issue
has been addressed.

https://bugzilla.gnome.org/show_bug.cgi?id=661762

--- data/ui/epiphany-ui.xml.orig	Mon Oct 17 18:46:51 2011
+++ data/ui/epiphany-ui.xml	Mon Oct 17 18:47:13 2011
@@ -13,7 +13,6 @@
 			<menuitem name="FilePrintMenu" action="FilePrint"/>
 			<menuitem name="FileSendToMenu" action="FileSendTo"/>
 			<separator name="FileSep3"/>
-			<menuitem name="FileWorkOfflineItem" action="FileWorkOffline"/>
 			<menuitem name="FileCloseWindowMenu" action="FileCloseTab"/>
 		</menu>
 
